packages/palette/src/elements/LineChart/LineChart.tsx

```diff
@@ -0,0 +1,166 @@
+import React, { useEffect, useRef, useState } from "react"
+import styled from "styled-components"
+import { media, space } from "../../helpers"
+import { ChartHoverTooltip } from "../DataVis/ChartHoverTooltip"
+import { ChartTooltipProps, coerceTooltip } from "../DataVis/ChartTooltip"
+import { ProvideMousePosition } from "../DataVis/MousePositionContext"
+import { useHasEnteredViewport } from "../DataVis/utils/useHasEnteredViewPort"
+import { Flex } from "../Flex"
+import { Sans } from "../Typography"
+import { LineChartSVG } from "./LineChartSVG"
+
+export interface PointDescriptor {
+  value: number
+  axisLabelX?: React.ReactNode
+  tooltip?: React.ReactNode | ChartTooltipProps
+}
+
+export interface LineChartProps {
+  points: PointDescriptor[]
+  chartHeight?: number
+}
+
+const margin = space(2)
+const DEFAULT_HEIGHT = 87
+
+/**
+ * LineChart is a component that displays some data points connected by lines.
+ * Useful for visualizing a time series, etc.
+ */
+export const LineChart: React.FC<LineChartProps> = ({
+  points,
+  chartHeight,
+}: LineChartProps) => {
+  const [width, setWidth] = useState(0)
+  const [height, setHeight] = useState(DEFAULT_HEIGHT)
```

Comment:
looks like you don't need `height` as state here? Can just use `chartHeight = DEFAULT_HEIGHT` in the props destructuring above.

Reply:
🙏 nice catch!